tests/New-DbaAgentSchedule.Tests.ps1

```diff
@@ -63,7 +63,8 @@ Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
             $jobId = (Get-DbaAgentJob -SqlInstance $script:instance2 -Job dbatoolsci_newschedule).JobID
             foreach ($key in $results.keys) {
                 $results[$key].EnumJobReferences() | Should -Contain $jobId
-                $results[$key].FrequencyTypes   | Should -BeIn $scheduleOptions
+                $results[$key].FrequencyTypes | Should -BeIn $scheduleOptions
+                $results[$key].JobCount | Should -Be 1
```

Comment:
This doesn't test the code block properly because you are calling Get command outside of the context of the command.

To properly test the count and add coverage this JobCount should be testing the output of the command and not via Get command.